src/python/director/transformUtils.py

```diff
@@ -157,28 +153,18 @@ def transformFromPose(position, quaternion):
     '''
     Returns a vtkTransform
     '''
-    rotationMatrix = np.zeros((3,3))
-    vtk.vtkMath.QuaternionToMatrix3x3(quaternion, rotationMatrix)
 
-    mat = np.eye(4)
-    mat[:3,:3] = rotationMatrix
+    mat = transformations.quaternion_matrix(quaternion)
     mat[:3,3] = position
-
-    t = vtk.vtkTransform()
-    t.SetMatrix(mat.flatten())
-    return t
+    return getTransformFromNumpy(mat)
 
 
 def poseFromTransform(transform):
     '''
     Returns position, quaternion
     '''
-    angleAxis = range(4)
-    transform.GetOrientationWXYZ(angleAxis)
-    angleAxis[0] = math.radians(angleAxis[0])
-    pos = transform.GetPosition()
-    quat = botpy.angle_axis_to_quat(angleAxis[0], angleAxis[1:])
-    return np.array(pos), np.array(quat)
+    mat = getNumpyFromTransform(transform)
+    return np.array(mat[:3,3]), transformations.quaternion_from_matrix(mat, isprecise=True)
```

Comment:
Previous method:  VTK transform to angle axis representation followed by botpy.angle_axis_to_quat
New method: call transformations conversion routine for matrix to quaternion.

The associated test confirms this provides the same answer for a randomly selected transform.